<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>523</width>
     <height>424</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QDialogButtonBox" name="settingsButtonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="settingsTabWidget">
      <property name="minimumSize">
       <size>
        <width>521</width>
        <height>391</height>
       </size>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="columnsWidget">
       <attribute name="title">
        <string>Columns</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>21</y>
          <width>345</width>
          <height>216</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="3">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="includeLabel_3">
             <property name="text">
              <string>Active Columns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="includeListWidget_3">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Accessed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Created</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Extension</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Filename</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Folder</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Matches</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modified</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Size</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="addPushButton_8">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addPushButton_11">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addPushButton_13">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="addPushButton_10">
             <property name="text">
              <string>Move Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addPushButton_12">
             <property name="text">
              <string>Move Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="limitsWidget">
       <attribute name="title">
        <string>Limits</string>
       </attribute>
       <widget class="QSpinBox" name="poolSpinBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
       <widget class="QLabel" name="poolLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Thread pool size:</string>
        </property>
        <property name="buddy">
         <cstring>poolSpinBox</cstring>
        </property>
       </widget>
       <widget class="QLabel" name="matchesLabel">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>90</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>[0: unlimited]</string>
        </property>
       </widget>
       <widget class="QLabel" name="linesLabel">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>120</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>[0: unlimited]</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop on total matches:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop on matching lines:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>281</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop folder recursion when folder depth exceeds value?</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>281</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start folder recursion when folder depth exceeds value?</string>
        </property>
       </widget>
       <widget class="QPushButton" name="defaultLimitsPushButton">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>330</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Defaults</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>59</y>
          <width>431</width>
          <height>101</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="title">
         <string>Grep</string>
        </property>
        <widget class="QSpinBox" name="matchesSpinBox">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>30</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>250</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="linesSpinBox">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>159</y>
          <width>371</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Find</string>
        </property>
        <widget class="QSpinBox" name="depthMaxSpinBox">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>20</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="depthMinSpinBox">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="recursionPushButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>260</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Disable folder recursion</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="presetsWidget">
       <attribute name="title">
        <string>Presets</string>
       </attribute>
       <widget class="QComboBox" name="targetComboBox">
        <property name="geometry">
         <rect>
          <x>64</x>
          <y>13</y>
          <width>421</width>
          <height>20</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>16</number>
        </property>
       </widget>
       <widget class="QLabel" name="targetLabel">
        <property name="geometry">
         <rect>
          <x>23</x>
          <y>13</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>232</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Target</string>
        </property>
        <property name="buddy">
         <cstring>targetComboBox</cstring>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>421</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
       <widget class="QRadioButton" name="skipRadioButton">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>40</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Skip</string>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="onlyRadioButton">
        <property name="geometry">
         <rect>
          <x>209</x>
          <y>40</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Only</string>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="alsoRadioButton">
        <property name="geometry">
         <rect>
          <x>67</x>
          <y>40</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Also</string>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup</string>
        </attribute>
       </widget>
       <widget class="QLabel" name="commandLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Command Line</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="commandLineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>461</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="binaryCheckBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>230</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Exclude &amp;binaries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cStyleCheckBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>270</y>
          <width>191</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Only search C-style function names</string>
        </property>
       </widget>
       <widget class="QPushButton" name="globalPushButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>330</y>
          <width>281</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit Global Presets</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="contentsRadioButton">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>By &amp;contents</string>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup_2</string>
        </attribute>
       </widget>
       <widget class="QRadioButton" name="extensionRadioButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>250</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>By &amp;extension then contents</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup_2</string>
        </attribute>
       </widget>
       <widget class="QPushButton" name="addPushButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>80</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="overridePushButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>300</y>
          <width>281</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Folder Overrides</string>
        </property>
       </widget>
       <widget class="QPushButton" name="defaultPresetsPushButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Defaults</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
